test/preflight/apps_v2_integration_test.go

```diff
@@ -419,7 +419,7 @@ func TestAppsV2Config_ProcessGroups(t *testing.T) {
 		return cmd
 	}
 
-	expectMachinesInGroups := func(machines []api.Machine, expected map[string]int) {
+	expectMachinesInGroups := func(machines []*api.Machine, expected map[string]int) {
```

Comment:
Because we define methods on `*api.Machine` not `api.Machine`?

Reply:
because that is what the main flyctl codebase does. no good reason, just that I was expecting the reference for all functions that take a list of machines